apps/ledger-live-desktop/README.md

```diff
@@ -15,11 +15,7 @@
 
 ## Architecture
 
-Ledger Live is an hybrid desktop application built with Electron, React, Redux, RxJS,.. and highly optimized with [ledger-core](https://github.com/LedgerHQ/lib-ledger-core) C++ library to deal with blockchains (sync, broadcast,..) via [ledger-core-node-bindings](https://github.com/LedgerHQ/lib-ledger-core-node-bindings). It communicates to Ledger hardware wallet devices (Nano X / Nano S / Blue) to verify address and sign transactions with [ledgerjs](https://github.com/LedgerHQ/ledger-live/tree/develop/libs/ledgerjs). Some logic is shared with [live-common](https://github.com/LedgerHQ/ledger-live/tree/develop/libs/ledger-live-common).
-
-<p align="center">
- <img src="./docs/architecture.png" width="550"/>
```

Comment:
can you also remove the file ./docs/architecture.png ? 👍

Reply:
Hello! Sure, I removed it. Not sure I did the right thing when I rebased with develop. Is it ok? Otherwise I will open a new PR.

Reply:
mmh probably open a new PR because there are a lot of unrelated changes indeed :'(

Reply:
No worries, next time I will get this right :') Sorry !